### PR TITLE
Added additional links for google

### PR DESCRIPTION
**Pull request by aditya28sarin**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Company data addition
- [ ] Bug fix
- [ ] New feature

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Browser:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
This looks great! The urls & display names are perfect.  One quick change needed: 
The line with `---` should be _after_ the additional_resources.  And a blank line at the end.

**Comment by aditya28sarin** [comment]
I have updated the changes that you wanted ma'am.

File: companies/google.md [hunk 1]
```diff
@@ -41,3 +41,14 @@ blm_statements:
 exec_team_url: https://blog.google/perspectives/
 diversity_inclusion_url: https://diversity.google/
 ---
```
Comment (kelseyhuse30) [inline companies/google.md:43]:
```suggestion
```

File: companies/google.md [hunk 2]
```diff
@@ -41,3 +41,14 @@ blm_statements:
...
+
```
Comment (kelseyhuse30) [inline companies/google.md:44]:
```suggestion
```

**Review by kelseyhuse30 — APPROVED** [review]

**Comment by kelseyhuse30** [comment]
Thanks @aditya28sarin !

**Pull request merged**